okrs/2019-q3.md

```diff
@@ -4,36 +4,36 @@
-| [Existing IPFS and IPLD documentation links to ProtoSchool tutorials where relevant](https://github.com/ProtoSchool/protoschool.github.io/issues/260)   | @fsdiogo  | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg)  |   |   |   |
-| [ProtoSchool supports tutorials containing multiple-choice exercises](https://github.com/ProtoSchool/protoschool.github.io/issues/177)| @terichadbourne | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg)|   |   | |
-| [ProtoSchool supports libp2p tutorials with validation](https://github.com/ProtoSchool/protoschool.github.io/issues/229)| @fsdiogo | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg)|   | | |
-| [Documentation for building tutorials includes pedagogical advice for lesson structure](https://github.com/ProtoSchool/protoschool.github.io/issues/253) |@terichadbourne|![p2](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p2.svg)||||
-| [A review of alternative online coding education platforms suggests whether we should incorporate an existing tool or continue to improve our custom system](https://github.com/ProtoSchool/protoschool.github.io/issues/264) | @fsdiogo  | ![p2](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p2.svg)  |   |     |   | |
+| All existing ProtoSchool tutorials link to additional documentation and learning materials both [in-line](https://github.com/ProtoSchool/protoschool.github.io/issues/262) and from new [resources pages](https://github.com/ProtoSchool/protoschool.github.io/issues/263) | @terichadbourne & @fsdiogo | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | 1.0 | 1.0 | |
+| [Existing IPFS and IPLD documentation links to ProtoSchool tutorials where relevant](https://github.com/ProtoSchool/protoschool.github.io/issues/260)   | @terichadbourne | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg)  | 0.9 | 1.0 |   |
+| [ProtoSchool supports tutorials containing multiple-choice exercises](https://github.com/ProtoSchool/protoschool.github.io/issues/177)| @terichadbourne | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg)| 1.0 | 1.0 | |
+| [Documentation for building tutorials includes pedagogical advice for lesson structure](https://github.com/ProtoSchool/protoschool.github.io/issues/253) |@terichadbourne|![p2](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p2.svg)| 0.0 | 1.0 ||
+| [A review of alternative online coding education platforms suggests whether we should incorporate an existing tool or continue to improve our custom system](https://github.com/ProtoSchool/protoschool.github.io/issues/264) | @fsdiogo  | ![p2](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p2.svg)  |  0.0 | 0.5 |   |
```

Comment:
As August 23rd is my last day, I'm not sure we can reach `0.5` on both this one and the non-MFS tutorial.